docs/source/tutorials/multifidelity/README.rst

```diff
@@ -17,4 +17,4 @@ problems.
    mf_asha
    mf_sync_model
    mf_async_model
-   mf_comparison
+   mf_comparison
```

Comment:
Can you also remove the files in multifidelity/img, or include them in the pasha tutorial?

Reply:
yes, I've moved them now